awscli/handlers.py

```diff
@@ -106,23 +132,25 @@ def awscli_initialize(event_handlers):
     # The s3 error message needs to registered before the
     # generic error handler.
     register_s3_error_msg(event_handlers)
-#    # The following will get fired for every option we are
-#    # documenting.  It will attempt to add an example_fn on to
-#    # the parameter object if the parameter supports shorthand
-#    # syntax.  The documentation event handlers will then use
-#    # the examplefn to generate the sample shorthand syntax
-#    # in the docs.  Registering here should ensure that this
-#    # handler gets called first but it still feels a bit brittle.
-#    event_handlers.register('doc-option-example.*.*.*',
-#                            param_shorthand.add_example_fn)
-    event_handlers.register('doc-examples.*.*',
-                            add_examples)
+    #    # The following will get fired for every option we are
+    #    # documenting.  It will attempt to add an example_fn on to
+    #    # the parameter object if the parameter supports shorthand
+    #    # syntax.  The documentation event handlers will then use
+    #    # the examplefn to generate the sample shorthand syntax
+    #    # in the docs.  Registering here should ensure that this
+    #    # handler gets called first but it still feels a bit brittle.
+    #    event_handlers.register('doc-option-example.*.*.*',
+    #                            param_shorthand.add_example_fn)
```

Comment:
Not relevant in the scope of the PR, but this comment block should probably be removed. It's been here for 10 years.